kratos/python/add_utilities_to_python.cpp

```diff
@@ -739,28 +740,69 @@ void AddUtilitiesToPython(pybind11::module& m)
+        .def("RemoveElementsAndBelongings", &Kratos::AuxiliarModelPartUtilities::RemoveElementsAndBelongings)
+        .def("RemoveElementAndBelongingsFromAllLevels", ModelPartRemoveElementAndBelongingsFromAllLevels1)
+        .def("RemoveElementAndBelongingsFromAllLevels", ModelPartRemoveElementAndBelongingsFromAllLevels2)
+        .def("RemoveElementAndBelongingsFromAllLevels", ModelPartRemoveElementAndBelongingsFromAllLevels3)
+        .def("RemoveElementAndBelongingsFromAllLevels", ModelPartRemoveElementAndBelongingsFromAllLevels4)
+        .def("RemoveElementsAndBelongingsFromAllLevels", &Kratos::AuxiliarModelPartUtilities::RemoveElementsAndBelongingsFromAllLevels)
+        .def("RemoveConditionAndBelongings", ModelPartRemoveConditionAndBelongings1)
+        .def("RemoveConditionAndBelongings", ModelPartRemoveConditionAndBelongings2)
+        .def("RemoveConditionAndBelongings", ModelPartRemoveConditionAndBelongings3)
+        .def("RemoveConditionAndBelongings", ModelPartRemoveConditionAndBelongings4)
+        .def("RemoveConditionsAndBelongings", &Kratos::AuxiliarModelPartUtilities::RemoveConditionsAndBelongings)
+        .def("RemoveConditionAndBelongingsFromAllLevels", ModelPartRemoveConditionAndBelongingsFromAllLevels1)
+        .def("RemoveConditionAndBelongingsFromAllLevels", ModelPartRemoveConditionAndBelongingsFromAllLevels2)
+        .def("RemoveConditionAndBelongingsFromAllLevels", ModelPartRemoveConditionAndBelongingsFromAllLevels3)
+        .def("RemoveConditionAndBelongingsFromAllLevels", ModelPartRemoveConditionAndBelongingsFromAllLevels4)
+        .def("RemoveConditionsAndBelongingsFromAllLevels", &Kratos::AuxiliarModelPartUtilities::RemoveConditionsAndBelongingsFromAllLevels)
+        ;
+
+    py::class_<TimeDiscretization::BDF1>(m, "BDF1")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF1::ComputeBDFCoefficients)
+        ;
+    py::class_<TimeDiscretization::BDF2>(m, "BDF2")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF2::ComputeBDFCoefficients)
+        ;
+    py::class_<TimeDiscretization::BDF3>(m, "BDF3")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF3::ComputeBDFCoefficients)
+        ;
+    py::class_<TimeDiscretization::BDF4>(m, "BDF4")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF4::ComputeBDFCoefficients)
+        ;
+    py::class_<TimeDiscretization::BDF5>(m, "BDF5")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF5::ComputeBDFCoefficients)
+        ;
+    py::class_<TimeDiscretization::BDF6>(m, "BDF6")
+        .def(py::init<>())
+        .def("ComputeBDFCoefficients", &TimeDiscretization::BDF6::ComputeBDFCoefficients)
+        ;
+
+    py::class_<TimeDiscretization::Newmark>(m, "Newmark")
+        .def(py::init<>())
+        .def("GetBeta", &TimeDiscretization::Newmark::GetBeta)
```

Comment:
No sets?

Reply:
no, they are computed

Reply:
how to you compute it? i would say that alpha and beta are input values

Reply:
gamma = 0.5 + alpha_m - alpha_f => returns 0.5 (for Newmark)
beta = 0.25(1 + alpha_m - alpha_f)^2 => returns 0.25 (for Newmark)

Reply:
see here: http://opensees.berkeley.edu/wiki/index.php/Generalized_Alpha_Method

Reply:
newmark has two parameters that can be changed (even though only one option gives second order...)